src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final String[] colors = new String[] {"red", "blue", "yellow", "green", "white"};

```

Comment:
```suggestion
    private static final String[] COLORS = new String[] {"red", "blue", "yellow", "green", "white"};

```

в именах констант пишутся все буквы большие https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names